From 2fa90048ebf41af71bbae20e9c5934286a049d75 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Sun, 16 Feb 2014 01:44:43 +0000
Subject: Fix window dragging on the path bar

GtkWindow relies on button press, release and motion notify
events being propagated. Therefore, we need to select for
motion notify events to make window dragging work.

https://bugzilla.gnome.org/show_bug.cgi?id=722542
---
diff --git a/src/nemo-pathbar.c b/src/nemo-pathbar.c
index 9d94f28..7d51d56 100644
--- a/src/nemo-pathbar.c
+++ b/src/nemo-pathbar.c
@@ -888,7 +888,8 @@
     attributes.event_mask |= 
         GDK_SCROLL_MASK |
         GDK_BUTTON_PRESS_MASK |
-        GDK_BUTTON_RELEASE_MASK;
+        GDK_BUTTON_RELEASE_MASK |
+        GDK_POINTER_MOTION_MASK;
     attributes_mask = GDK_WA_X | GDK_WA_Y;
 
     path_bar->event_window = gdk_window_new (gtk_widget_get_parent_window (widget),

